geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako

```diff
@@ -11,7 +11,7 @@ services:
 ${service_defaults('config')}\
 
   print:
-    image: camptocamp/mapfish_print:3.12.1
+    image: camptocamp/mapfish_print:3.14
```

Comment:
Should be 3.14.1, no?

Reply:
Not if you want to take bug fixes automatically and have daily rebuilds.

Reply:
no because we want the daily updates